src/thumbnailer.h

```diff
@@ -57,32 +57,41 @@ class Thumbnailer {
                         // the byte budget.
     kStatsError,        // In case of error while getting frame's size and PSNR.
     kWebPMuxError,      // In case of error related to WebPMux object.
-    kSlopeOptimError    // In case of error while using slope optimization to
+    kSlopeOptimError,   // In case of error while using slope optimization to
                         // generate animation.
+    kGenericError       // For other errors.
   };
 
+  enum Method { kEqualQuality = 0, kEqualPSNR, kSlopeOptim };
+  static constexpr Method MethodList[] = {kEqualQuality, kEqualPSNR,
```

Comment:
By inlining it I meant something similar to
```c++
for (const libwebp::Thumbnailer::Method& method :
     {kEqualQuality, kEqualPSNR, kSlopeOptim}) {
  ...
}
```
and get rid of `MethodList` but this is just a suggestion. Otherwise call it `kMethodList` to respect the constant naming convention.

Reply:
It's 
```for (const libwebp::Thumbnailer::Method& method :
       {libwebp::Thumbnailer::kEqualQuality, libwebp::Thumbnailer::kEqualPSNR,
        libwebp::Thumbnailer::kSlopeOptim}) {
```
which I think is kind of messy. I'll rename it as a constant.